code/modules/client/preference_setup/loadout/lists/headwear.dm

```diff
@@ -28,7 +28,7 @@
 /datum/gear/head/beret/bsec
 	display_name = "beret, Operative"
 	path = /obj/item/clothing/head/beret/sec/navy/officer
-	allowed_roles = list("Ironhammer Operative","Ironhammer Commander","Gunnery Sergeant","Ironhammer Medical Specialist")
```

Comment:
Lieutenant and Sergeant gets their own berets.

Reply:
yes? that didn't change
![image](https://user-images.githubusercontent.com/112724221/194016567-7786beb1-7095-4f41-8dae-dd457ee38c1e.png)

Reply:
what did he mean by this........................................